packages/flutter_tools/templates/create/lib/main.dart.tmpl

```diff
@@ -55,6 +58,19 @@ class MyHomePage extends StatefulWidget {
 
 class _MyHomePageState extends State<MyHomePage> {
   int _counter = 0;
+{{#withPluginHook}}
+  String _platformVersion = 'Unknown';
+
+  @override
+  void initState() {
+    super.initState();
+    {{pluginDartClass}}.platformVersion.then((String platformVersion) {
+      setState(() {
+        _platformVersion = platformVersion;
```

Comment:
This code is buggy. If the _MyHomePageState state has been disposed before the future completes, it'll assert.

Is there a way to cancel a pending platform request? We should do so in the state's `dispose()` function so that it doesn't unexpectedly complete here in that case.